sessionctx/variable/tidb_vars.go

```diff
@@ -198,6 +201,7 @@ const (
 	DefTiDBMemQuotaNestedLoopApply   = 32 << 30 // 32GB.
 	DefTiDBGeneralLog                = 0
 	DefTiDBRetryLimit                = 10
+	DefTiDBDisableTxnAutoRetry       = false
```

Comment:
I think it could be true as default.

Reply:
This should be false by default for now. Because many users rely on the auto-retry.